docs/docs/external-volumes

```diff
@@ -0,0 +1,79 @@
+     "mem": 32,   
+     "cpus": 0.1,   
+     "instances": 1,
+     "env": {
+          "DVDI_VOLUME_DRIVER": "rexray",
+          "DVDI_VOLUME_NAME": "test12345",
+          "DVDI_VOLUME_OPTS":"size=1,volumetype=standard,newfstype=ext4,overwritefs=false"
+     }
+}
+```
+
+In the app definition above, the environment variables (`env`) specify a 1GB volume that will be formatted ext4 once it is created.
+
+- Volumes will always be available under `$REXRAY_HOME/var/lib/rexray/volumes/$DVDI_VOLUME_NAME`. **Note:** This applies only to non-Docker apps; Docker apps can specify the mount point inside the app definition. See [the REX-Ray documentation on data directories](https://rexray.readthedocs.org/en/v0.3.2/user-guide/config/#data-directories) for more information.
+
+- The root path inside the volume for storing data is configurable, but defaults to `data/`. See [the REX-Ray documentation on root paths](https://rexray.readthedocs.org/en/v0.3.2/user-guide/config/#volume-root-path-031) for more information.
+
+- `DVDI_VOLUME_DRIVER` tells `DVDCLI` which Docker volume driver we want to use for storage. If you are running Marathon on DCOS, this value should always be `rexray`. [Learn more about REX-ray](https://rexray.readthedocs.org/en/v0.3.2/user-guide/schedulers/).
+
+- `DVDI_VOLUME_NAME` is the name by which REX-Ray looks up your volume. When your task is staged on an agent, REX-Ray queries the storage service for a volume with this name. If one does not exist, it's created. Otherwise, the existing volume is re-used.
+
+- `DVDI_VOLUME_OPTS` is the set of volume options passed to DVDCLI. [See the full list of options](https://github.com/emccode/dvdcli/blob/3c3dcbe1490c2c4a150555978f2b4915dafaddf2/README.md#extra-options).
+
+You can define multiple volumes per app by repeating the environment variables with 0, 1 ,2, etc. appended to them. For example, `DVDI_VOLUME_NAME0`.
+
+**Note:** Most of the configuration of your external volume is cannot be changed after you create it. The exceptions are `overwriteFs` and `newFsType`. If `overwriteFs` is true, REX-Ray will reformat the volume every time it is mounted on an agent with the filesystem specified by `newFsType`. If `overwriteFs` is false, howeverm REX-Ray will only format the volume when it is created.
+
+### Docker apps
+
+Refer to the [REX-Ray documentation](https://rexray.readthedocs.org/en/v0.3.2/user-guide/schedulers/#docker-containerizer-with-marathon) for an example of an app definition of a Docker app that uses external volumes.
+
+## Potential Pitfalls
+
+- Your storage provider may have particular limitations. For instance, on Amazon's EBS, you can only assign one task per volume.
+
+- The volumes you create are not automatically cleaned up. If you you delete your cluster, go to your storage provider and delete the volumes you no longer need. Find them by searching by `DVDI_VOLUME_NAME`.
```

Comment:
The user should search whatever interface/UI the storage provider offers, which is why it's hard to be exact here. Using EBS as an example, you'd go to the AWS EC2 console, click on Volumes, and search by the `DVDI_VOLUME_NAME` that was set in the Marathon app, which will be the value of the `Name` tag on the volume.